src/bo4e/zusatzattribut.py

```diff
@@ -24,7 +24,5 @@ class ExterneReferenz(COM):
 
     """
 
-    ex_ref_name: Optional[
-        str
-    ] = None  #: Bezeichnung der externen Referenz (z.B. "microservice xyz" oder "SAP CRM GP-Nummer")
-    ex_ref_wert: Optional[str] = None  #: Wert der externen Referenz (z.B. "123456"; "4711")
+    name: Optional[str] = None  #: Bezeichnung der externen Referenz (z.B. "microservice xyz" oder "SAP CRM GP-Nummer")
+    wert: Optional[Any] = None  #: Wert der externen Referenz (z.B. "123456"; "4711")
```

Comment:
Der Name eines Zusatzattributs sollte eigentlich niemals `None` sein. Mir fällt jedenfalls keine Situation ein, in der das irgendwie sinnvoll wäre.
`Optional[Any]` ist dasselbe wie `Any` 😉

Reply:
fixed in 15f1286b025f3bfc2c165046f9d9fd573fdc6608